ComposeTesting/README.md

```diff
@@ -36,17 +38,24 @@
 
 ## API reference
 
+For API reference information for **testing-kotlin** , refer to these resources:
 
-### FoldingFeature Helper
+- [UiDevice Extensions](https://github.com/microsoft/surface-duo-sdk/tree/main/utils/test-utils#uidevice-extensions)
+- [FoldingFeatureHelper](https://github.com/microsoft/surface-duo-sdk/tree/main/utils/test-utils#foldingfeaturehelper)
+- [DeviceModel](https://github.com/microsoft/surface-duo-sdk/tree/main/utils/test-utils#devicemodel)
+- [WindowLayoutInfoConsumer](https://github.com/microsoft/surface-duo-sdk/tree/main/utils/test-utils#windowlayoutinfoconsumer)
+- [CurrentActivityDelegate](https://github.com/microsoft/surface-duo-sdk/tree/main/utils/test-utils#currentactivitydelegate)
+- [ForceClick](https://github.com/microsoft/surface-duo-sdk/tree/main/utils/test-utils#forceclick)
+- [ViewMatcher](https://github.com/microsoft/surface-duo-sdk/tree/main/utils/test-utils#viewmatcher)
 
-These functions can be used in foldable UI tests to simulate the present of vertical and
-horizontal foldingFeatures(folds/hinges). The foldingFeatures are simulated using TestWindowLayoutInfo, using the Google [Jetpack WindowManager Testing](https://developer.android.com/reference/androidx/window/testing/layout/package-summary) library.
+In addition to the resources above, **ComposeTesting** also offers the APIs described below.
 
-```kotlin
-fun createWindowLayoutInfoPublisherRule(): TestRule
-```
+### FoldingFeatureHelper
```

Comment:
Can we have some information about the "import"s? I think I missed `.compose.` in our samples too. I will fix it, but it may be a little bit confused if the developer didn't pay attention and would like to move from the old ComposeTesting to the new one.
```
import com.microsoft.device.dualscreen.testing.compose.getString
import com.microsoft.device.dualscreen.testing.compose.simulateHorizontalFoldingFeature
import com.microsoft.device.dualscreen.testing.compose.simulateVerticalFoldingFeature
import com.microsoft.device.dualscreen.testing.createWindowLayoutInfoPublisherRule
```

Reply:
Good catch, I just added a text block - let me know if that works!